assets/js/modules/analytics/components/dashboard/AnalyticsDashboardWidget.js

```diff
@@ -123,168 +127,148 @@ class AnalyticsDashboardWidget extends Component {
+	const vAxes = null;
 
-		const {
-			dateRange,
-		} = this.props;
+	// Hide Analytics data display when we don't have data.
+	const wrapperClass = ! loading && receivingData ? '' : 'googlesitekit-nodata';
+	const currentDateRange = getCurrentDateRange( dateRange );
 
-		const series = this.buildSeries();
-		const vAxes = null;
-
-		// Hide Analytics data display when we don't have data.
-		const wrapperClass = ! loading && receivingData ? '' : 'googlesitekit-nodata';
-		const currentDateRange = getCurrentDateRange( dateRange );
-
-		return (
-			<Fragment>
-				<Header />
-				<Alert module="analytics" />
-				<div className="googlesitekit-module-page googlesitekit-module-page--analytics">
-					<div className="mdc-layout-grid">
-						<div className="mdc-layout-grid__inner">
-							<div className="
+	return (
+		<Fragment>
+			<Header />
+			<Alert module="analytics" />
+			<div className="googlesitekit-module-page googlesitekit-module-page--analytics">
+				<div className="mdc-layout-grid">
+					<div className="mdc-layout-grid__inner">
+						<div className="
 								mdc-layout-grid__cell
 								mdc-layout-grid__cell--span-12
 							">
-								<PageHeader
-									title={ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
-									icon
-									iconWidth="24"
-									iconHeight="26"
-									iconID="analytics"
-									status="connected"
-									statusText={ sprintf(
-										/* translators: %s: module name. */
-										__( '%s is connected', 'google-site-kit' ),
-										_x( 'Analytics', 'Service name', 'google-site-kit' )
-									) }
+							<PageHeader
+								title={ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
+								icon
+								iconWidth="24"
+								iconHeight="26"
+								iconID="analytics"
+								status="connected"
+								statusText={ sprintf(
+									/* translators: %s: module name. */
+									__( '%s is connected', 'google-site-kit' ),
+									_x( 'Analytics', 'Service name', 'google-site-kit' )
+								) }
+							/>
+							{ loading && <ProgressBar /> }
+						</div>
+						{ /* Data issue: on error display a notification. On missing data: display a CTA. */ }
+						{ ! receivingData && (
+							error ? getDataErrorComponent( 'analytics', error, true, true, true, errorObj ) : getNoDataComponent( _x( 'Analytics', 'Service name', 'google-site-kit' ), true, true, true )
+						) }
+						<div className={ classnames(
+							'mdc-layout-grid__cell',
+							'mdc-layout-grid__cell--span-12',
+							wrapperClass
+						) }>
+							<Layout
+								header
+								/* translators: %s: date range */
+								title={ sprintf( __( 'Audience overview for the last %s', 'google-site-kit' ), currentDateRange ) }
+								headerCtaLabel={ sprintf(
+									/* translators: %s: module name. */
+									__( 'See full stats in %s', 'google-site-kit' ),
+									_x( 'Analytics', 'Service name', 'google-site-kit' )
+								) }
+								headerCtaLink="http://analytics.google.com"
```

Comment:
This piece is something that was missing from the ACs I just noticed. This should also link to `https://analytics.google.com/analytics/web/#/report/visitors-overview/a{accountID}w{internalWebPropertyID}p{profileID}/`, like the similar items on the main dashboard do.